src/rastervision/common/data/generators.py

```diff
@@ -250,3 +264,14 @@ def transform_img_batch(x):
         split_gen = map(transform_img_batch, img_batch_gen)
 
         return split_gen
+
+    def normalize_constants(self, datasets_path):
```

Comment:
This name feels a bit off...it took me a bit of reading the code to see what it was doing. Can we come up with a better name? `write_normalization_constants`? Maybe `write_normalization_params` , as that jives more with the `normalization_params` that are loaded from this file. I can see that there's a differentiation between const vs params, but I'd say that if this is simply a serialization of the normalization params, we can name it all `normalization_params`